crates/node-core/src/args/rpc_server_args.rs

```diff
@@ -176,6 +176,14 @@ pub struct RpcServerArgs {
     /// Gas price oracle configuration.
     #[command(flatten)]
     pub gas_price_oracle: GasPriceOracleArgs,
+
+    /// Enable auth engine api over IPC  
+    #[arg(long, default_value_if("dev", "true", Some("true")))]
+    pub auth_ipc: bool,
+
+    /// Filename for auth IPC socket/pipe within the datadir
+    #[arg(long = "auth-ipc.path", default_value = constants::DEFAULT_ENGINE_API_IPC_ENDPOINT)]
+    pub auth_ipc_path: Option<String>,
```

Comment:
okay, in this case we don't need an option, because what decides if we want to enable this is  `auth_ipc`
```suggestion
    pub auth_ipc_path: String,
```